TLS.md

```diff
@@ -37,21 +37,21 @@ To manually run a Redis server with TLS mode (assuming `gen-test-certs.sh` was
 invoked so sample certificates/keys are available):
 
 For TLS built-in mode:
-    ./src/redis-server --tls-port 6379 --port 0 \
+    ./src/valkey-server --tls-port 6379 --port 0 \
         --tls-cert-file ./tests/tls/redis.crt \
         --tls-key-file ./tests/tls/redis.key \
```

Comment:
We might need to change redis.crt and redis.key to valkey files in tls mode.

Reply:
Yes, but there are other PRs changing this. Let's try to avoid doing everything in one PR.

Reply:
Got you!